docs/src/apimanual.md

```diff
@@ -446,6 +446,12 @@ An important note for the LP case is that the signs of the feasible duals depend
 
 Currently, a convention for duals is not defined for problems with non-conic sets ``\mathcal{S}_i`` or quadratic functions ``f_0, f_i``.
 
+#### Duality and scalar product
+
+The scalar product is different from the canonical one for the sets [PositiveSemidefiniteConeTriangle](@ref), [LogDetConeTriangle](@ref), [RootDetConeTriangle](@ref).
+For these cones, the nonzero entries of the matrix ``A_i`` of the section [Duals](@ref) may be different from the `coefficients` field in the `VectorAffineFunction`.
```

Comment:
"may be different" is ambiguous. Say exactly which are different and which are not.